frontend/.env.example

```
@@ -9,10 +9,17 @@ APP_URL=
 DO_CDN_URL=
 
 NEXT_PUBLIC_POSTHOG_KEY=
+POSTHOG_API_KEY=
 NEXT_PUBLIC_POSTHOG_HOST=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_POSTHOG_HOST key should go before the NEXT_PUBLIC_POSTHOG_KEY key

(UnorderedKey)